.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: release
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+
+  upload-transifex:
+
+    name: Upload pot files to transifex
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
```

Comment:
Can you add a comment here that states that the installation steps are modeled after the `docs` job as part of the `continuous-integration` workflow?